### PR TITLE
Remove shooter class items from initial items

### PR DESCRIPTION
**Pull request by MinetestSam**

### CODE REVIEW

File: abandoned_isles/map.conf [hunk 1]
```diff
@@ -8,6 +8,5 @@ rotation = x
 h = 140
 team.2 = blue
 name = Abandoned Isles
-initial_stuff = default:sword_steel,shooter:pistol,default:pick_steel,default:sandstone 50,default:torch 10,shooter:ammo
-r = 115
```
Comment (ClobberXD) [inline abandoned_isles/map.conf:12]: Please revert this change

File: abandoned_isles/map.conf [hunk 2]
```diff
@@ -8,6 +8,5 @@ rotation = x
 h = 140
 team.2 = blue
 name = Abandoned Isles
-initial_stuff = default:sword_steel,shooter:pistol,default:pick_steel,default:sandstone 50,default:torch 10,shooter:ammo
```
Comment (ClobberXD) [inline abandoned_isles/map.conf:11]: Why has `shooter:ammo` been removed?

Comment (MinetestSam) [reply abandoned_isles/map.conf:11]: I thought it was a part of shooter class, my bad

Comment (ClobberXD) [reply abandoned_isles/map.conf]: It would be a part of the upcoming sniper class, but there's no guarantee that the PR would be merged anytime soon. It's advisable to not pre-emptively code in anticipation of a future change.
